### PR TITLE
Simple not enough funds

### PR DESCRIPTION
**Pull request by zerogamma**

### CODE REVIEW

**Review by omniwired — APPROVED** [review]

File: packages/screens/DAppStore/apps/toripunks/content/RussianRoulette.tsx [hunk 1]
```diff
@@ -177,6 +181,15 @@ export const Russian = () => {
       return setResult(!result);
     }
     if (!bet) return;
+    if (balance.length > 0) {
+      if (
+        balance[0].denom === "utori" &&
```
Comment (hthieu1110) [inline packages/screens/DAppStore/apps/toripunks/content/RussianRoulette.tsx:186]: if user does not have utori then we by pass this check ? 
According to this check, we check only if denom = utori, is that expected behaviour ?

Comment (omniwired) [reply packages/screens/DAppStore/apps/toripunks/content/RussianRoulette.tsx:186]: yes, the toripunks app only cares about TORI

File: packages/screens/DAppStore/apps/toripunks/content/RussianRoulette.tsx [hunk 2]
```diff
@@ -177,6 +181,15 @@ export const Russian = () => {
...
+        parseInt(balance[0].amount, 10) < bet * 1000000
```
[the remark this answers is not in the document]
Comment (omniwired) [reply packages/screens/DAppStore/apps/toripunks/content/RussianRoulette.tsx:187]: won't fix

**Review by hthieu1110 — APPROVED** [review]
LGTM

**Comment by zxxma** [comment]
LGTM

**Pull request merged**